config-templates/ibm/ibm-vpc-block-csi-driver/5.1/storage-class-parameters.json

```diff
@@ -0,0 +1,155 @@
+[
+    {
```

Comment:
Please compare this file with 5.0 version file. Lot of regex being removed. Please review